pkg/scheduler/core/generic_scheduler.go

```diff
@@ -272,7 +272,13 @@ func (g *genericScheduler) processPreemptionWithExtenders(
 				var err error
 				// Replace nodeToVictims with result after preemption from extender.
 				if nodeToVictims, err = extender.ProcessPreemption(pod, nodeToVictims, g.cachedNodeInfoMap); err != nil {
-					return nil, err
+					if extender.IsIgnorable() {
+						glog.Warningf("Skip: extender: %v is currently unavailable: %v",
```

Comment:
IIUC, this skips even if the extender is available but returns an error. While this is not what I had in my mind when I created issue, this solves the problem.

I think this error could be misleading. We could have gotten some other error as well from extender, while continuing part seems fine, we should  let the actual error be shown here. Smthg like 

```go
glog.Warningf("Skipping extender %v as it returned error %v and has ignorable flag set", extender, err)
```

Reply:
Got it, make sense.